clients/src/main/java/org/apache/kafka/clients/tools/VerifiableProducer.java

```diff
@@ -124,6 +132,23 @@ private static ArgumentParser argParser() {
                 .choices(0, 1, -1)
                 .metavar("ACKS")
                 .help("Acks required on each produced message. See Kafka docs on request.required.acks for details.");
+        
+        parser.addArgument("--producer-prop")
+                .action(Arguments.append())
+                .required(false)
+                .type(String.class)
+                .dest("userSpecifiedProducerProps")
+                .metavar("PRODUCER-PROP")
+                .help("Any custom producer properties of the form key=val.");
+
+        parser.addArgument("--close-timeout")
+                .action(store())
+                .required(false)
+                .setDefault(10L)
+                .type(Long.class)
+                .metavar("CLOSE-TIMEOUT")
+                .dest("closeTimeoutSeconds")
+                .help("When SIGTERM is caught, wait at most this many seconds for unsent messages to flush before stopping the VerifiableProducer process.");
```

Comment:
Generally configs for anything time related are in ms. I think there may be an exception or two when the time scale is expected to be much larger (log rolling, log retention have flexible units). Not sure if we want to try to be as consistent as possible (use ms) or use the units that most likely match what the user will want to express (use seconds).